_posts/Getting Started/2020-08-14-Configure-off-cluster-pravega-clients.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+category: "Getting Started"
+tags: [flink, java, SDP, Pravega]
+subtitle: "Configure off-cluster Pravega clients on Dell EMC Streaming Data Platform"
+img: pravega-client.png
+license: Apache
+support: Community
+author: 
+    name: Youmin Han
+    description: Nautilus App Developer
+    image: batman.png
+css: 
+js: 
+---
+This is a general guide for configuring Pravega applications/clients to connect to SDP when the client is running off-cluster.
+<!--more-->
+
+## Purpose
+Before ingesting data into a Pravega stream on Dell EMC Streaming Data Platform, you need to configure off-cluster Pravega clients for the secruity verify. The purpose of this guide is to demonstrate configuring keycloak authentication, retrieving Pravega controller external IP, and setting environment variables.
```

Comment:
add a paragraph that talks about how, when a Pravega client is running on-cluster in Kubernetes, authentication is injected automatically by SDP operators.  However, when a Pravega client runs off-cluster, in order to connect to an SDP stream, Keycloak authentication must be configured properly.